### ISSUE TITLE
Serialization

### ISSUE THREAD

**Issue by GeTechG** [rj00a/evenio#58]
Can you make an example of serialization using `evenio`, and is it generally possible?

**Comment by andrewgazelka** [comment]
@GeTechG are you wanting to serialize the state of a `World` using serde or something?

**Comment by GeTechG** [comment]
> are you wanting to serialize the state of a `World` using serde or something?

@andrewgazelka Yes, I would like to be able to do something like what is in hecs, having a context where you can decide how to serialize a particular entity.
